### PR TITLE
Create a "Login Page" for Angular

### PR DESCRIPTION
**Pull request by TheGuyWhoCodes**
## Majors
 - Adds a "Login" page. This login page has a simple button to login, then redirects over to the Google oAuth login system
 - Does this by calling login endpoint to get login oAuth endpoint, then redirects to whatever it gets from the API
 - In the future this should get a check to make sure we aren't already logged in.
## Minors
 - Adds creation test for /login
## Screenshots
image

### CODE REVIEW

File: mediaphile/src/app/pages/login/login.component.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="login">
+  <div class="login-triangle"></div>
+
+  <h2 class="login-header">Log in to Mediaphile</h2>
+
+  <div class="login-container">
+    <a (click)="login()"> <img class = "login-button" src="assets/sign-in.png"></a>
```
Comment (henryksloan) [inline mediaphile/src/app/pages/login/login.component.html:7]: No space around `=` in attributes

File: mediaphile/src/app/pages/login/login.component.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+})
+export class LoginComponent implements OnInit {
+
+  constructor(private infoSvc: InfoService) { }
+
+  ngOnInit(): void {
+  }
+
+  public login() {
+    this.infoSvc.login().subscribe(x => {
```
Comment (henryksloan) [inline mediaphile/src/app/pages/login/login.component.ts:17]: Rename `x` to something descriptive like loginStatus.

Comment (TheGuyWhoCodes) [reply mediaphile/src/app/pages/login/login.component.ts]: Fixed per 368cfb8

File: mediaphile/src/app/pages/login/login.component.html (hunk 1 above)
Comment (TheGuyWhoCodes) [reply mediaphile/src/app/pages/login/login.component.html]: Fixed per 368cfb8

**Pull request merged**